tests/ui.rs

```diff
@@ -98,8 +114,89 @@ fn miri_config(target: &str, path: &str, mode: Mode, with_dependencies: bool) ->
         ..Config::rustc(path)
     };
 
+    if let Mode::Pass { rustc: true } = mode {
+        config.comment_defaults.base().add_custom("run", Run {
+            exit_code: 0,
+            output_conflict_handling: Some(|path, actual, errors, config| {
+                let path = path.with_file_name(
```

Comment:
```suggestion
                // ui_test would compare the output with `test.run.stdout`, but we want to
                // compare with `test.stdout`, so adjust the filename.
                let path = path.with_file_name(
```
At least I think that's what happens?